Gemfile

```diff
@@ -1,12 +1,7 @@
 source "https://rubygems.org"
 
-group :development do
-  gem 'bundler', '>= 1.0'
-  gem 'rake', '10.5.0'
-  gem 'minitest', '~> 5.8', '>= 5.8.4'
-  gem 'pry', '>= 0.10.1'
-  gem 'travis', '>= 1.8.2'
-  gem 'jeweler', '>=2.0.1'
-  gem 'codeclimate-test-reporter'
-  gem 'simplecov'
-end
+gemspec
+
+gem "minitest"
```

Comment:
Any reason for dropping the `group :development do`?

I'd prefer keeping it, but maybe there is a good reason for dropping?

Reply:
In a gem, the `Gemfile` file is always used for development, so it is not necessary to differentiate by groups and they can go to the Bundler default group (`:default`) and thus the `Gemfile` file is much simpler.